src/test/model_service_test.cpp

```diff
@@ -313,3 +313,224 @@ TEST(RestModelStatus, serialize2Json) {
     ASSERT_EQ(error_status, StatusCode::OK);
     EXPECT_EQ(json_output, expected_json);
 }
+
+const ovms::ModelConfig DUMMY_MODEL_WITH_ONLY_NAME_CONFIG{
+    "dummy",
+};
+
+TEST(ModelService, getAllModelsStatuses_one_model_one_version) {
```

Comment:
Please add 2 negative tests for:
static Status getAllModelsStatuses(std::map<std::string, tensorflow::serving::GetModelStatusResponse>& models_versions, ModelManager& manager);
    static Status serializeModelsStatuses2Json(const std::map<std::string, tensorflow::serving::GetModelStatusResponse>& models_versions, std::string& output);